packages/phrases/src/locales/en/translation/admin-console/enterprise-sso-details.ts

```diff
@@ -24,6 +24,8 @@ const enterprise_sso_details = {
   connector_logo_field_description: 'Each image should under 500KB, SVG, PNG, JPG, JPEG only.',
   branding_logo_context: 'Upload logo',
   branding_logo_error: 'Upload logo error: {{error}}',
+  branding_light_logo_context: 'Upload light mode logo',
+  branding_light_logo_error: 'Upload light mode logo error: {{error}}',
```

Comment:
nit:
1. It seems we don't need to emphasis a logo is for light mode since it's still the default settings. So I think we only need to say: `Upload logo` here.
2. The error message can be improved a little bit: `Error on uploading logo: {{error}}`

Reply:
1. it's by design, when the dark mode logo uploader is enabled, emphasis "light" mode logo uploader to distinguish it from the dark mode one sound reasonable to me.

Reply:
ok